pytradfri/api/libcoap_api.py

```diff
@@ -149,7 +149,6 @@ def generate_psk(self, security_key):
         if not self._psk:
             existing_psk_id = self._psk_id
 
-            self._psk_id = 'Client_identity'
```

Comment:
@lwis I believe this is the reason users may have experienced problems. I got this back from the gateway when debugging. We would expect 9090 to be pytradfri in current version and uuid.uuid4().hex in proposed update of the lib.
```
{'9090': 'Client_identity'}
['coap-client', '-u', 'Client_identity', '-k', 'PnHhjOjepj8vhbZB', '-v', '0', '-m', 'post', '-f', '-', 'coaps://192.168.0.129:5684/15011/9063']
```

Reply:
Clarification: the output above is from before I removed the line.

Reply:
You are right in that Client_identity user generates psk but we also expect 9090 to contain the identity we want a psk for: https://twitter.com/home_assistant?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor

It works as expected in the aiocoap api but not in the libcoap api.

Reply:
Is the request method broken?

Reply:
I'm unsure, libcoap seems to work fine if the key has already been generated and is available in the file (json or in the gateway_psk.txt file.)